### ISSUE TITLE
Patreon status update?

### ISSUE THREAD

**Issue by caron** [ocornut/imgui#1156]
So what is the status of supporting this project/you through Patreon?

**Comment by ocornut** [comment]
Hello Steven,

Thanks for querying about it. The Patreon (https://www.patreon.com/imgui) has been disabled since a few months because I was too busy releasing my game to spend enough time on the library.  The game is now out and I'm working on Steam and a few other versions but soon after I intend to get back to dear imgui, and will re-enable the Patreon. 

I was hoping to enable it for the June 1st milestone but I think it'll be July 1st, aka I will resume work from June. You can already/still add yourself to the monthly Patreon, it just won't be taken off your account until the first month I enable it.

Note that in the event your workplace is interested in contributing to imgui, it is possible to avoid Patreon all-together and can you more formal invoices that can be billed through my company.

I would like to make dear imgui with full-time job this year but this will only work if I can gather serious funding from companies (also see https://github.com/ocornut/imgui/wiki/Future-features). Ideally if serious funding came we could drop the Patreon. So far I just recently gathered 1 month worth of funding from one major studio. However if this idea doesn't work out long term, the Patreon would allow me to keep spending some time on the library while pursuing another job.

TL;DR; if you are an individual Patreon is great. Otherwise contact me :)

-Omar

**Comment by Pagghiu** [comment]
I think it's worth to say that  to support the project a regular company could buy some software developement services from @ocornut and receive proper invoice, just like we did last year.
Patreon doesn't work well for our fiscal system: with the regular invoice we can shave some local taxes and makes the process a little more clean for our system.

To @ocornut: you can count on a software development "support" from us for _at least_ the same amount as last year. We know it's small but if our business goes better we will be more than happy to support more!! 😄 

I really encourage every company to try doing the same, it would be a dream to have Omar collect enough money to be able spending sizable amount of time improving the library!

**Comment by caron** [comment]
@ocornut thanks for the update, my interest is personal so I would make use of Patreon.
